meta-mender-toradex-nxp/recipes-bsp/u-boot/u-boot-toradex-fsl-fw-utils_%.bbappend

```diff
@@ -0,0 +1 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```

Comment:
Hmm, no I can't say I've seen this before.

I don't fully trust poky/master these days though, after the huge "recipe specific sysroots" feature was merged upstream there has been a lot of small issues. Might be worth temporarily reverting to poky/master from a couple of weeks ago, and see if it's still a problem.